packages/scxa-cell-type-wheel/src/CellTypeWheel.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import HighchartsReact from 'highcharts-react-official'
+import Highcharts from 'highcharts'
+import highchartsSunburst from 'highcharts/modules/sunburst'
+
+highchartsSunburst(Highcharts)
+
+// SUPER HACK!
+// https://github.com/highcharts/highcharts/issues/4994
+window.Highcharts = Highcharts
+
+class CellTypeWheel extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      // To avoid unnecessary update keep all options in the state.
```

Comment:
Sorry I did not get this comment, whey do we need to set those options as state? I think we could utilize a stateless component here.